components/apprentice-status-indicator/apprentice-status-indicator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './apprentice-status-indicator.module.scss';
+
+const ApprenticeStatusIndicator = ({ status }) => {
+  if (!status) return null;
+
+  if (status === 'current') {
+    return (
+      <svg id="triangle" viewBox="0 0 100 100" className={styles['employee-status']}>
+        <polygon
+          points="50 15, 100 100, 0 100"
+          className={styles['current-employee-status']}
+          data-testid="current"
+        />
+      </svg>
+    );
+  }
+
+  if (status === 'previous') {
```

Comment:
If we update this to use a single test ID, then the only thing that changes here is the `className`. This could then be refactored to return a single svg element and just use a variable to determine the appropriate `className`, similar to your previous implementation.